index.d.ts

```diff
@@ -7,6 +7,8 @@ declare module "mockingcase" {
     random?: boolean;
     onlyLetters?: boolean;
     firstUpper?: boolean;
+    upper?: string | object;
```

Comment:
```suggestion
    upper?: string | RegExp;
```

Reply:
> Also, can you double check the declaration file. Not sure if I did that right.

I made the changes from my comment  and successfully tested in a typescript project